manpages/pihole.8

```diff
@@ -260,9 +260,16 @@ Available commands and options:
       #m                Disable Pi-hole functionality for # minute(s)
 .br
 
-\fBrestartdns\fR
+\fBrestartdns\fR [options]
 .br
-    Restart Pi-hole subsystems
+    Full restart Pi-hole subsystems
+.br
+
+    (restart options):
+.br
+      reload            Updates the lists, flushes dnsmasq 's DNS cache
```

Comment:
It may be a bit misleading to write `dnsmasq` here. Just omit it.
```suggestion
      reload            Updates the lists and flushes DNS cache
```